landing-strip/core/src/main/java/com/novoda/landingstrip/FixedWidthTabsContainer.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.landingstrip;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+public class FixedWidthTabsContainer extends TabsContainer {
```

Comment:
how about a TabsContainerFactory - no need for inheritance as neither of the two implementations do anything different/take different params?

`TabViewsContainer` may also make it explicit that the class is meant to hold tab views, and then discourage people from adding behaviour that doesn't belong there - in my understanding, it was meant as a guard against the LinearLayout to only allow certain operations through, like `addTabView()` -> `ll.addView()`, `clear()` -> `ll.removeAllViews()`, etc.